docs/docs/developers/contracts/references/portals/outbox.md

```diff
@@ -30,45 +34,33 @@ Allows a recipient to consume a message from the `Outbox`.
 
 | Name           | Type        | Description |
 | -------------- | -------     | ----------- |
-| `_message`     | `L2ToL1Msg` | The message to consume |
-| ReturnValue    | `bytes32`   | The hash of the message | 
+| `_l2BlockNumber`     | `uint256` | The block number specifying the block that contains the message we want to consume |
+| `_leafIndex`     | `uint256` | The index inside the merkle tree where the message is located |
+| `_message`     | `L2ToL1Msg` | The L2 to L1 message |
```

Comment:
Should we have the message first? Not sure if that makes a nice separation, e.g., `message, {proof}` is essentially the setup then